### ISSUE TITLE
Notifications get delayed after uptimes bigger than a day.

### ISSUE THREAD

**Issue by Adrianrff** [wasdennnoch/AndroidN-ify#1210]
I'm not really sure if this is an N-ify related issue or something else, but I thought I'd post here just in case. 

If the phone hasn't been rebooted in like a day or more, notifications start to appear in the status bar several seconds after they arrive (sound is played). This happens with any notification affected by N-ify, including phone calls when the screen is on. Rebooting fixes the problem momentarily, but since I'm experiencing an issue (**number 1016, reported here**) with N-ify and Xnotification, I'd like avoid having to reboot too often.

[xposed_error_20161114_110823.txt](https://github.com/wasdennnoch/AndroidN-ify/files/589660/xposed_error_20161114_110823.txt)

If this is not related to N-ify, my apologies. 

Thanks in advance.

**Comment by pwrusr** [comment]
Under n-ify settings...enable "Fix sound and Notification tile"  > reboot (:

**Comment by Adrianrff** [comment]
Yeah... I seriously doubt that has something to do with my problem. Thanks though.

**Comment by pwrusr** [comment]
If ur issue got solved.. Please close it (:

**Comment by Adrianrff** [comment]
@pwrusr ???

I'll close it when it's either solved or I get a useful answer from one of the devs.

**Comment by wasdennnoch** [comment]
@pwrusr That's completely unelated.
@Adrianrff I'll have a look but I have the issue that the whole device gets laggy after some days and not just the notifications so I may not find an answer. You can test if the problem persists with N-ify disabled.

**Comment by pwrusr** [comment]
Sorry.. For being unrelated... I should add I think label (; Kidding....i havnt saw his issue completely... I felt like short essay on my device... Jk (;.. I saw some sound and Notification related issue..here and in nify.... I never played with sound and Notification tile in nify. . Soo I thought it might worth a shot... Or do solve his problem....but I misunderstood it... Good to see u back with some fixes (:

**Comment by Adrianrff** [comment]
@wasdennnoch Alright. Thanks. As I said, I'm  not sure if it's even related  to N-ify, but I just thought I make an entry and see if other people were experiencing something similar. Since we're here, what about the issue 1016? I'm certainly not the only one having it, although it's labled as not reproducible. The Xnotification folks are also aware of it and said they're having a look.

**Comment by wasdennnoch** [comment]
The only quick reply "issues" others are having are missing inputs because of multiple messages or apps that don't support it. In your case it appears to be totally random and we haven't seen that anywhere else.
